packages/flutter_test/lib/src/controller.dart

```diff
@@ -217,6 +316,309 @@ class SemanticsController {
+    performAction(
+      node,
+      SemanticsAction.moveCursorBackwardByWord,
+      args: shouldModifySelection);
+  }
+
+  /// Performs a [SemanticsAction.setText] action on the [SemanticsNode]
+  /// found by `finder` using the given `text`.
+  ///
+  /// Throws a [StateError] if:
+  /// * The given `finder` returns zero or more than one result
+  /// * The [SemanticsNode] found with `finder` does not support
+  /// [SemanticsAction.setText]
+  void setText(Finder finder, String text) {
+    final SemanticsNode node = findWithAction(finder, SemanticsAction.setText);
+    performAction(node, SemanticsAction.setText, args: text);
+  }
+
+  /// Performs a [SemanticsAction.setSelection] action on the [SemanticsNode]
+  /// found by `finder`.
+  ///
+  /// The `base` parameter is the start index of selection, and the `extent`
+  /// parameter is the length of the selection. Each value should be limited
+  /// between 0 and the length of the found [SemanticsNode]'s `value`.
+  ///
+  /// Throws a [StateError] if:
+  /// * The given `finder` returns zero or more than one result
+  /// * The [SemanticsNode] found with `finder` does not support
+  /// [SemanticsAction.setSelection]
+  void setSelection(Finder finder, {required int start, required int end}) {
+    final SemanticsNode node = findWithAction(finder, SemanticsAction.setSelection);
+    performAction(
+      node,
+      SemanticsAction.setSelection,
+      args: <String, int>{'base': start, 'extent': end},
+    );
+  }
+
+  /// Performs a [SemanticsAction.copy] action on the [SemanticsNode]
+  /// found by `finder`
+  ///
+  /// Throws a [StateError] if:
+  /// * The given `finder` returns zero or more than one result
+  /// * The [SemanticsNode] found with `finder` does not support
+  /// [SemanticsAction.copy]
+  void copy(Finder finder) {
+    final SemanticsNode node = findWithAction(finder, SemanticsAction.copy);
+    performAction(node, SemanticsAction.copy);
+  }
+
+  /// Performs a [SemanticsAction.cut] action on the [SemanticsNode]
+  /// found by `finder`
+  ///
+  /// Throws a [StateError] if:
+  /// * The given `finder` returns zero or more than one result
+  /// * The [SemanticsNode] found with `finder` does not support
+  /// [SemanticsAction.cut]
+  void cut(Finder finder) {
+    final SemanticsNode node = findWithAction(finder, SemanticsAction.cut);
+    performAction(node, SemanticsAction.cut);
+  }
+
+  /// Performs a [SemanticsAction.paste] action on the [SemanticsNode]
+  /// found by `finder`
+  ///
+  /// Throws a [StateError] if:
+  /// * The given `finder` returns zero or more than one result
+  /// * The [SemanticsNode] found with `finder` does not support
+  /// [SemanticsAction.paste]
+  void paste(Finder finder) {
+    final SemanticsNode node = findWithAction(finder, SemanticsAction.paste);
+    performAction(node, SemanticsAction.paste);
+  }
+
+  /// Performs a [SemanticsAction.didGainAccessibilityFocus] action on th
```

Comment:
missing `e` at end of line

Reply:
Well, _that's_ a weird one. Must have over-selected while editing or something.

Good catch, fixed!